book/types_of_data.md

```diff
@@ -362,29 +360,83 @@ You can also create a table as a list of records, JSON-style:
 ╰───┴──────┴──────╯
 ```
 
-:::tip
-Internally, tables are simply **lists of records**. This means that any command which extracts or isolates a specific row of a table will produce a record. For example, `get 0`, when used on a list, extracts the first value. But when used on a table (a list of records), it extracts a record:
+To access a row or a column, you can use the [`get`](/commands/docs/get.md) command or cell path access:
```

Comment:
You may also want to mention that `get <nth>` gets a row whereas `get <name>` gets a column. Technically, you can have columns named <nth> as well but when you `get` them you have to enclose them in quotes. e.g. `get "5"` would get the column named `"5"` instead of the row named `5`. Columns are always strings; rows are always numbers.